caching/store.go

```diff
@@ -20,8 +20,8 @@ func (ds *DBStore[K, V]) Get(idx int) (*btree.Node[K, int, V], error) {
 		return nil, err
 	}
 	node := &btree.Node[K, int, V]{}
-	err = msgpack.Unmarshal(bytes, node)
-	return node , nil
+	_ = msgpack.Unmarshal(bytes, node)
+	return node, nil
```

Comment:
this is the wrong fix, we should return err not hide it.

Reply:
I agree, but I also believe this is probably something we should do in many other places.

Reply:
Resolving this comment, see https://github.com/PlakarKorp/plakar/pull/317#issuecomment-2619062380